swift/Ice/SwiftUI-greeter/ContentView.swift

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) ZeroC, Inc.
+
+import SwiftUI
+
+struct ContentView: View {
+    @EnvironmentObject var client: GreeterClient
+    @State private var host = "localhost"
+    @State private var user = "SwiftUI"
+    @State private var greeting = ""
+    @State private var error: String?
+    @State private var isLoading = false
+
+    var body: some View {
+        Form {
+            Section {
+                HStack {
+                    Text("Host:")
+                    TextField("e.g. localhost", text: $host)
```

Comment:
Not sure that we need the extra Text field. Maybe just `TextField("Host:", ...` and ` TextField("Name:",...`
```suggestion
                    Text("Host:")
                    TextField("e.g. localhost", text: $host)
```

Reply:
Is that common on iOS, the TextField first argument only shows up when empty doesn't it?  I added the label to make clear what the input is about, I'm fine dropoing it...